### PR TITLE
Add missing fixtures for new Figgy IDs.

### PR DESCRIPTION
**Pull request by tpendragon**
These tests are doing live queries against Figgy. It's a real problem.

### CODE REVIEW

**Comment by tpendragon** [comment]
More and more tests are failing while Figgy's converting IDs, we'll have to take care of this next week.

File: spec/fixtures/alma/current_fixtures.json [hunk 1]
```diff
@@ -16452,5 +16452,695 @@
...
+      "format":[
+        "Book"
+      ],
+      "electronic_access_1display":[
+        "{\"http://arks.princeton.edu/ark:/88435/np193c809\":[\"arks.princeton.edu\",\"EXKA Ptolemy 1525\"],\"http://arks.princeton.edu/ark:/88435/kw52jc00n\":[\"arks.princeton.edu\",\"EXKA Ptolemy 1525a\"]}"
+      ],
+      "electronic_access_index":[
+        "http://arks.princeton.edu/ark:/88435/np193c809 EXKA Ptolemy 1525",
+        "http://arks.princeton.edu/ark:/88435/kw52jc00n EXKA Ptolemy 1525a"
+      ],
```
Comment (christinach) [inline spec/fixtures/alma/current_fixtures.json:16546]: @tpendragon I'm trying to replicate the fix you did in this pr. In solr electronic_access_1_display has more information indexed while you are adding only what is in 856. There is no electronic_access_index field. Why did you add it?

Comment (christinach) [reply spec/fixtures/alma/current_fixtures.json:16546]: ok I see the key `iiif_manifest_paths` needs to be removed from the electronic_access_1display.
